extensions/amp-viewer-integration/0.1/messaging.js

```diff
@@ -71,12 +70,18 @@ export class WindowPortEmulator {
   /**
    * @param {!Window} win
    * @param {string} origin
+   * @param {!Window} target
+   * @param {string=} opt_logsId for dev logging.
```

Comment:
It's fine to have `opt_logsId` in `AmpViewerHost`, but we shouldn't need it for `WindowPortEmulator`. Let's remove.